sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB_impl.hpp

```diff
@@ -122,7 +122,8 @@ int main() {
                        float>();
       }
     }
-    if (combinations[i].atype == matrix_type::sint8) {
+    if (combinations[i].atype == matrix_type::sint8 &&
```

Comment:
I noticed 2 combinations satisfy this condition, so the same tests are running 2 times on PVC, hence added one more condition, so only 1 combination satisfies the condition.